TeamCode/src/main/java/com/acmerobotics/relicrecovery/drive/MecanumDrive.java

```diff
@@ -19,58 +22,52 @@
  *      |       |
  * (2)//---------\\(3)
  *
- * the paper: http://www.chiefdelphi.com/media/papers/download/2722
+ * the paper: http://www.chiefdelphi.com/media/papers/download/2722 (see doc/Mecanum_Kinematic_Analysis_100531.pdf)
  */
 public class MecanumDrive {
```

Comment:
In general, I think we should come up with a good convention for units in this class. It's quite easy to mix up units currently. I think it would be good to have `getPositions()` and friends return the same units as those required by `getDelta()`

Reply:
that sounds like a good idea

Reply:
Ok. From now on, `getPositions()` and friends will return radians

Reply:
In the end, I actually decided to create a separate set of methods (`getRotations()`, etc.) for returning radians. This way we can maintain consistency with the SDK (e.g. `DcMotor#getCurrentPosition()`). I've also made sure to document the units for everything.